models/aliyun_ftasr_api_zh/SBI

```diff
@@ -8,11 +8,29 @@ fi
 scp=$1
 dir=$2
 oss_scp=${scp}_oss
+part_nums=10
 
 # download ossutil64 tools
 wget -O ossutil64 https://gosspublic.alicdn.com/ossutil/1.7.13/ossutil64
 chmod 755 ossutil64
 
+# upload wav to oss
 ./upload_wav.py $scp $oss_scp
-./asr_sdk.py $oss_scp $dir/raw_rec.txt
+
+# split test set 
+awk -vf=${part_nums} -vl="`wc -l $oss_scp`" 'BEGIN{p=int(l/f);q=(l%f);for(n=1;n<=f;n++)a[n]=n*p+((n<=q)?++x:x)}{if(NR>a[i])i++;print > "'$oss_scp'."i}' $oss_scp
```

Comment:
Here you append $oss_scp  with split indexes as temporary paths, according to spec, you are not allowed to write temp files outside "result dir" (i.e. $dir). please put them explicitly inside $dir